config-example.yaml

```diff
@@ -0,0 +1,5 @@
+api_base_url: '/v1/staff-fee-privilege'
+term_id: '201701'
+osu_id: '123456789'
```

Comment:
Could we get the osu_id and term_id dynamically during the tests? I'm thinking of being able to get a bunch of IDs in the /staff-fee endpoint, then making a /staff-fee/{id} request to each ID.